projectDocs/dev/remoteDebugging.md

```diff
@@ -0,0 +1,126 @@
+
+If you prefer, you can set up the tunnel [at the command line instead][CLI create tunnel].
+
+* `code tunnel` to create a session tunnel.
+* `code tunnel service install` to create a background service tunnel.
+
+### On the development machine
+
+1. Open VS Code.
+2. Open the Remote Menu (`Ctrl`+`Alt`+`O`) and choose "Connect to Tunnel...".
+   * If you do not have the Remote-Tunnels extension installed, select "Tunnel" to install it.
+3. Choose the account type you used to publish the tunnel from your secondary machine.
+   If you're not already logged in, log in when prompted.
+   Ensure you log in to the same account you used to publish the tunnel.
+4. Select the tunnel that you created from your secondary machine.
+   VS Code will let you know when you have connected.
+   * The first time you connect, this may take some time as VS Code downloads necessary components.
+   * Even if you have the required extensions installed locally, VS Code may prompt you to install them in the remote workspace.
+     If prompted, proceed with the installation of the extensions.
+5. Go to the Explorer view (`Ctrl`+`Shift`+`E`), select "Open Folder" and browse to the NVDA repository on the secondary machine (or type its path).
+6. VS Code on your development machine is now acting as a remote control for your secondary machine.
+   Anything you do now will be carried out on your secondary machine.
+7. With an appropriate `launch.json` (such as that in the NVDA VS Code workspace configuration) in the secondary machine's `.vscode` directory in the NVDA repository, you can now run and debug NVDA.
+   NVDA will run on the secondary machine, and you can interact with the Debug view on the development machine.
+
+If you prefer, you can also debug the remote copy of NVDA in a browser by entering the vscode.dev URL that was generated on the secondary machine.
+
+### When you're done
+
+When you're done, it's a good idea to disconnect from the remote tunnel on your development machine, and close the tunnel on your secondary machine.
+
+* On your development machine, make sure you've saved any changes you have made, then open the Remote menu (`Ctrl`+`Alt`+`O`) and choose "Close Remote Connection".
+* On your secondary machine, open the Command Palette (`Ctrl`+`Shift`+`P`) and run "Remote Tunnels: Turn off Remote Tunnel Access...".
+  Press "Yes" when prompted.
+  * If you created a session tunnel, closing VS Code without turning off remote tunnel access will close the tunnel until you open VS Code again.
+  * If you created the tunnel as a service, the tunnel will continue to remain active in the background until you turn off remote tunnel access.
+  * If you created the tunnel at the CLI, use `Ctrl`+`C` to close it.
+    If you created it as a service, run `code tunnel kill` to close the tunnel, or `code tunnel service uninstall` to remove it.
+
+## Example session: debug the date/time command
+
+1. Set up your development and secondary machines as above.
```

Comment:
```suggestion
1. Set up your development and secondary machines as above, so that your development machine is now controlling your secondary machine.
```